code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -621,6 +621,22 @@
 		W.charges = W.max_charges
 	update_icon()
 
+/obj/item/storage/belt/wands/fireballs
+	name = "infernal belt"
+	desc = "\"Use Fireball and only Fireball. Nothing but fireball. Just Fireball. <b>Just Fireball.</b>\""
```

Comment:
Why do we put this in quotation marks while the other descs aren't?

Reply:
Its a quote